### ISSUE TITLE
Running navigation stack and Gmapping in On-board raspberry Pi

### ISSUE THREAD

**Issue by Zeonsilt06** [XRobots/ReallyUsefulRobot#2]
Hi james. Can i ask you something about this project. Do you run all the Gmapping node and Navigation stack all on Raspberry Pi onboard or some of them in remote computer with SSH ? i also built the same system like you now, but can't figure out why the map update keep missing it's desired rate. I suspect some computation issue, but seeing your video with the same setting as me, it looks all good.

**Comment by XRobots** [comment]
No, it's running on a remote workstation. I get that as well, apparently there's a known bug somewhere.

**Comment by Zeonsilt06** [comment]
oh so you do the Gmapping Slam and navigation stack in remote workstation , but all serial node and rplidar node is in raspberry pi. Thank you very much !

**Comment by XRobots** [comment]
Yep. I did put some notes in the readme about what runs where.

**Comment by Zeonsilt06** [comment]
do you have performance issue running node through SSH, like Lidar data delay ?

**Comment by XRobots** [comment]
Not that I know of - it seems to work ok
